scripts/core/loader.yml

```diff
@@ -225,3 +225,23 @@ ordinal: "1"
 params: []
 returns:
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: fptr_typedef
+desc: "Callback function to retrieve output from the logger."
+name: $x_logger_output_callback_t
+return: "void"
+params:
+    - type: const char*
+      name: pLoggerMsg
+      desc: "[in][out] pointer to data to be passed to callback"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Set a callback function for use by the logger to retrieve logging output."
+class: $xLoaderConfig
+loader_only: True
+name: SetLoggerCallback
+decl: static
+params:
+    - type: $x_logger_output_callback_t
+      name: pfnLoggerCallback
+      desc: "[in] Function pointer to callback from the logger."
```

Comment:
There should always be a `void * user_data` argument which gets passed into the callback to let the user access any context they need.

Reply:
I've added this now. It's passed all the way through to the logger sink not sure if that's correct?